test/slow/api_spec_slow.js

```diff
@@ -184,23 +194,20 @@ describe(`electron-forge API (with installer=${installer.substr(12)})`, () => {
         expect(normalPackageJSON).to.have.deep.property('config.forge');
       });
 
-      let targets = [];
-      if (fs.existsSync(path.resolve(__dirname, `../../src/makers/${process.platform}`))) {
-        targets = fs.readdirSync(path.resolve(__dirname, `../../src/makers/${process.platform}`)).map(file => path.parse(file).name);
-      }
-      const genericTargets = fs.readdirSync(path.resolve(__dirname, '../../src/makers/generic')).map(file => path.parse(file).name);
-
-      [].concat(targets).concat(genericTargets).forEach((target) => {
-        describe(`make (with target=${target})`, () => {
-          before(async () => {
-            const packageJSON = await readPackageJSON(dir);
-            packageJSON.config.forge.make_targets[process.platform] = [target];
-            await fs.writeFile(path.resolve(dir, 'package.json'), JSON.stringify(packageJSON));
-          });
-
-          it('successfully makes with default config', async () => {
-            await forge.make({ dir, skipPackage: true });
-          });
+      const target = 'zip';
```

Comment:
Got this locally, in a manner that will port well to the downstream changes (i.e. importing the dummy maker)